topics/community/tutorials/community-lab/tutorial.md

```diff
@@ -67,61 +67,39 @@ communities/<your community>/lab/sections_templates/
         5_support_and_help.yml
         6_community.yml
 
-{% include _includes/cyoa-choices.html option1="GitHub_Actions" option2="Command-line" default="GitHub_Actions"
-       text="The files can be created in two wyas, using GitHub actions (on the GitHub interface) or using the command line to launch a script." %}
-
-<div class="GitHub_Actions" markdown="1">
+The files are created and updated automatically using GitHub actions, you just need to add a bit of code to point to your community folder.
 
 > <hands-on-title>Generate the files using Github Actions</hands-on-title>
 >
 > 1. Go to the [Galaxy Codex repo](https://github.com/galaxyproject/galaxy_codex)
-> 2. Go to the file `.github/workflows/populate_labs.yaml`
+> 2. Go to the file `.github/workflows/fetch_filter_resources.yaml`
 > 3. On the right, click on the pen to "Edit this file"
-> 4. Duplicate section called "Populate biodiversity Lab"
+> 4. Scroll down
+> 5. Duplicate the section called "Populate <community-name> Lab"
 >
->    For example:
 >    ```
->    - name: Populate biodiversity Lab
->      run: | 
->        bash sources/bin/populate_labs.sh
->      env: 
->        COMMUNITY: biodiversity
+>      - name: Populate <community-name> Lab
+>        run: |
+>          bash sources/bin/populate_labs.sh
+>        env: 
+>          COMMUNITY: <community-name>
 >    ```
-> 5. Change the name and community name for your community
-> 6. Commit changes the changes to a new branch that you name "new lab for <Community-name>"
-> 7. On the top panel of the Galaxy codex repository, click on "Actions"
-> 8. On the left panel, click on "Populate labs"
-> 9. In the table, click on "Run workflow" and select the branch you just created "new lab for <Community-name>"
-> 10. Click on "Run workflow"
->
-> This will create a Pull Request with the new files in the appropriate community folder. To find the Pull Request, on the top panel of the Galaxy codex repository, click on "Pull requests".
-{: .hands_on}
-
-</div>
-
-<div class="Command-line" markdown="1">
-
-> <hands-on-title>Generate the files using the command line</hands-on-title>
->
-> 1. Fork and clone the [Galaxy Codex repo](https://github.com/galaxyproject/galaxy_codex)
-> 2. Set the `COMMUNITY` variable : `export $COMMUNITY=<your community>`
-> 3. Launch `sources/bin/populate_labs.sh`
+> 6. Replace <community-name> by the name of your community
```

Comment:
Add some backticks around <community> otherwise it will be interpreted as html (and nothing is shown)

```suggestion
> 6. Replace `<community-name>` by the name of your community
```